### PR TITLE
Gitis Settlement Area

### PR DESCRIPTION
**Pull request by terrapin47**

### CODE REVIEW

File: QUEST_LV_0200.tsv [hunk 1]
```diff
@@ -3827,53 +3827,53 @@ QUEST_LV_0200_20150323_003826	$These are the research documents that I asked fro
...
 QUEST_LV_0200_20150323_003868	These were made for plant-type monsters so they would also influence the crops for sure.{nl}It was my mistake that I didn't think about it deeply.
 QUEST_LV_0200_20150323_003869	If it weren't for you, we could have spoiled the harvest which we put enourmous effort into.{nl}They should grow well now right?
-QUEST_LV_0200_20150323_003870	They are late. I mean, my colleagues who went to Ishigiti herb fields to defeat Gorgon.{nl}I don't wanna think that they were annihilated.
+QUEST_LV_0200_20150323_003870	They are late. I mean, my colleagues who went to the Ishigiti Herb Farm to defeat the Gorgon.{nl}I don't wanna think that they were annihilated.
 QUEST_LV_0200_20150323_003871	Soldier Purse
 QUEST_LV_0200_20150323_003872	{memo X}Run! Gorgon is coming!
 QUEST_LV_0200_20150323_003873	I hope I was wrong.{nl}The monsters are going crazy..
-QUEST_LV_0200_20150323_003874	You defeated Gorgon youself?{nl}It is fortunate that my colleagues are safe, but I don't know what they did as soldiers..{nl}
+QUEST_LV_0200_20150323_003874	You defeated the Gorgon youself?{nl}It is fortunate that my colleagues are safe, but I don't know what they did as soldiers..{nl}
```
Comment (Sunaries) [inline QUEST_LV_0200.tsv:3874]: Youself -> yourself

File: QUEST_LV_0200.tsv [hunk 2]
```diff
@@ -5242,18 +5242,18 @@ QUEST_LV_0200_20150323_005241	$Gitis told you that the workers at the settlement
 QUEST_LV_0200_20150323_005242	$The Pyromancer Master told you that she will definitely help you. Talk to the Pyromancer Master again.
 QUEST_LV_0200_20150323_005243	$Collect the samples of Pink Chupalukas at the settlement land
 QUEST_LV_0200_20150323_005244	{memo X}Pyromancer Master told you that she will definitely help you. First get some samples of Pink Chupalukas at the settlement land.
-QUEST_LV_0200_20150323_005245	$Hand it over to the Pyromancer Master
+QUEST_LV_0200_20150323_005245	$Bring the items to the Pyromancer Master
 QUEST_LV_0200_20150323_005246	$You've obtained the samples of Pink Chupalukas. Hand them over to the Pyromancer Master.
-QUEST_LV_0200_20150323_005247	Obtain %s from the settlement land
+QUEST_LV_0200_20150323_005247	Obtain %s from Ziuluti Vacant Lot
```
Comment (Sunaries) [inline QUEST_LV_0200.tsv:5247]: Is this an error by IMC? Original says 개척지 (=Settlement Area, even though the full name should be Gitis Settlement Area, 기티스의 개척지)

Comment (terrapin47) [inline QUEST_LV_0200.tsv:5247]: Could be, you do find them in the Gitis Settlement Area, but the marker is at Ziuluti Vacant Lot, and the quests always tell you what Area already.

**Comment by Sunaries** [comment]
Good to merge. Well done.

**Comment by terrapin47** [comment]
Resolved conflicts.

**Comment by imcgames** [comment]
:+1:

**Pull request merged**